catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/converter/mgmp/MgmpConverter.java

```diff
@@ -0,0 +1,1289 @@
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.data.types.Associations;
+import ddf.catalog.data.types.Contact;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.data.types.DateTime;
+import ddf.catalog.data.types.Location;
+import ddf.catalog.data.types.Media;
+import ddf.catalog.data.types.Topic;
+
+/**
+ * Not thread safe.
+ */
+public class MgmpConverter extends AbstractGmdConverter {
+
+    public static final String RESOURCE_SECURITY_CODE_LIST_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/resourceConstraints/mgmp:MGMP_SecurityConstraints/classification/MD_ClassificationCode/@codeList";
+
+    public static final String RESOURCE_ORIGINATOR_SECURITY_CODE_LIST_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/resourceConstraints/mgmp:MGMP_SecurityConstraints/mgmp:originatorClassification/MD_ClassificationCode/@codeList";
+
+    public static final String RESOURCE_STATUS_CODE_LIST_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/status/MD_ProgressCode/@codeList";
+
+    public static final String GMD_MIN_ALTITUDE_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/verticalElement/EX_VerticalExtent/minimumValue/gco:Real";
+
+    public static final String GMD_VERTICAL_CRS_XLINK_HREF_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/verticalElement/EX_VerticalExtent/verticalCRS/@xlink:href";
+
+    public static final String GMD_VERTICAL_CRS_XLINK_TITLE_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/verticalElement/EX_VerticalExtent/verticalCRS/@xlink:title";
+
+    public static final String ASSOCIATIONS_RELATED_CODE_SPACE_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/aggregationInfo/MD_AggregateInformation/aggregateDataSetIdentifier/RS_Identifier/codeSpace/gco:CharacterString";
+
+    public static final String ASSOCIATIONS_RELATED_TYPE_CODE_LIST_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/aggregationInfo/MD_AggregateInformation/associationType/DS_AssociationTypeCode/@codeList";
+
+    public static final String ASSOCIATIONS_RELATED_TYPE_CODE_LIST_VALUE_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/aggregationInfo/MD_AggregateInformation/associationType/DS_AssociationTypeCode/@codeListValue";
+
+    public static final String ASSOCIATIONS_RELATED_TYPE_TEXT_PATH =
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/aggregationInfo/MD_AggregateInformation/associationType/DS_AssociationTypeCode";
+
+    public static final String SECURITY_METADATA_CLASSIFICATION_ATTRIBUTE =
```

Comment:
The MetacardType now uses these constants in `Security.java` :  https://github.com/codice/alliance/blob/master/catalog/core/catalog-core-api/src/main/java/org/codice/alliance/catalog/core/api/types/Security.java